src/routes/(app)/admin[[hash=admin_hash]]/picture/[id]/+page.svelte

```diff
@@ -28,6 +34,28 @@
 	</label>
 
 	<Picture picture={data.picture} class="object-contain max-h-[500px] max-w-full" />
+	{#each data.picture.storage.formats as format, i}
+		<code
+			>{`${data.websiteLink}/picture/raw/${data.picture?._id}/format/${format.width}`}
```

Comment:
Actually @ithiame we should not need `data.websiteLink` at all in the frontend, it should be removed from the codebase.

We can use `$page.url.origin`, let's at least use it here.

Reply:
ok - see my PR review comment for work remaining:


> I think that there should still be a way for users to download digital files associated to no products?
> 
> Maybe we can add a secret key to each digital file uploaded, and give a link with that secret key in the url?
> 
> Then the admin can share that link, and users can use it to download the file